### PR TITLE
docs(appium): recognize headspin as development partner

### PR DESCRIPTION
**Pull request by jlipps**
Starting to go live with some of our partner/sponsor announcements

### CODE REVIEW

File: packages/appium/docs/overrides/partials/footer.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+<div class="appium-sponsor-thanks">
+  <h3>With thanks to our Development Partner</h3>
+  <div style="text-align: center">
+    <a href="https://www.headspin.io/solutions/appium-mobile-test-automation">
+      <img src="/docs/en/assets/images/sponsor-logo-headspin.png" style="width: 200px;" />
+    </a>
+  </div>
+</div>
+
+<!-- Footer -->
```
Comment (jlipps) [inline packages/appium/docs/overrides/partials/footer.html:12]: note that to create a custom footer, i had to create an override file and copy in the existing default footer code to augment it.

**Review by eglitise — COMMENTED** [review]
I will try to build the docs tomorrow and take a look. I assume the development partners are visible in the footer of all docs pages?

File: README.md [hunk 2]
```diff
@@ -178,6 +178,27 @@ cross-platform native app automation, for mobile and beyond!
...
 documentation on [How Does Appium Work?](https://appium.io/docs/en/latest/intro/appium/).
 
+## Sponsors
+
+Appium has a [Sponsorship Program](GOVERNANCE.md#sponsorship)! If you or your company uses Appium
+and wants to give back financially to the project, we use these funds to [encourage development and
+contributions](GOVERNANCE.md#compensation-scheme), as well as support other open source projects we
+rely on. [Become a sponsor](https://opencollective.com/appium) via our OpenCollective page.
+
+### Development Partner
```
[the remark this answers is not in the document]
Comment (jlipps) [reply README.md:188]: this was an intentional choice; once we have more than one i would update it to plural?

**Comment by jlipps** [comment]
> I will try to build the docs tomorrow and take a look. I assume the development partners are visible in the footer of all docs pages?

yes, that's right. it looks pretty good. i wish there was a way to not show it on the index page (since sponsor logos are already there), but didn't have time to figure out whether that is possible or not.

**Review by KazuCocoa — APPROVED** [review]
lgtm

**Comment by eglitise** [comment]
[I've made a few changes to the footer](https://github.com/appium/appium/compare/jlipps/dev-partner-announce...eglitise:appium:dev-partner-announce) that I think make it cleaner and easier to extend - thoughts?

**Comment by jlipps** [comment]
ok @eglitise I applied your changes and suggestions, thanks they look great!

**Review by eglitise — APPROVED** [review]

**Pull request merged**